cookbook/bundles/tests.rst

```diff
@@ -0,0 +1,311 @@
+.. index::
+   single: Bundle; Testing third party bundles
+
+Writing Tests for Third Party Bundles
+=====================================
+
+The Standard Edition's front controller looks like this::
+
+    // web/app.php
+    use Symfony\Component\HttpFoundation\Request;
+
+    require_once __DIR__.'/../vendor/autoload.php';
+    require_once __DIR__.'/../app/AppKernel.php';
+
+    // Create the request from PHP variables super globals such as $_GET, $_POST, $_SERVER, etc
+    $request = Request::createFromGlobals();
+
+    // The application handles the Request
+    $kernel = new AppKernel('prod', false);
+    $response = $kernel->handle($request);
+
+    // Dump the Response's headers and print the Response's content
+    $response->send();
```

Comment:
I'd remove this block - it's a bit of a maintenance problem. Instead, just describe how you could create a kernel and a "fake" request in order to test your bundle.